### PR TITLE
RST-144 Regenerate language files

### PR DESCRIPTION
**Pull request by robyoung**
The language files were not correctly regenerated after removing the 'forward email' section from the emails in #310

This change regenerates the language files and updates the `makemessages.sh` script to regenerate both the English and the Welsh files.

### CODE REVIEW

File: conf/locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -2699,6 +2685,12 @@ msgstr "Questions about your pleas?"
 msgid "Need help using this service?"
 msgstr "Need help using this service?"
 
+#~ msgid "do not reply to it or click any links"
```
Comment (leoapost) [inline conf/locale/en/LC_MESSAGES/django.po]: are these commented out entries to be used in the future?

Comment (robyoung) [inline conf/locale/en/LC_MESSAGES/django.po]: no, good spot, I'll remove them

Comment (leoapost) [inline conf/locale/en/LC_MESSAGES/django.po]: I see more in the `conf/locale/cy/LC_MESSAGES/django.po` file

Comment (robyoung) [inline conf/locale/en/LC_MESSAGES/django.po]: I've pushed the change. It's also highlighted that I've missed some changes from #312 leaving `# TODO` markers in. I'll add them to this PR and change the title.

File: conf/locale/cy/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -2431,10 +2431,9 @@ msgstr "sicrhau bod eich dirwy yn seiliedig ar eich amgylchiadau"
 msgid "help to speed up the processing of your case"
 msgstr "helpu i gyflymu'r broses o ran hwyluso eich achos"
 
-# TODO
 #: apps/result/management/commands/process_results.py:96
 msgid "Make a plea result"
-msgstr "[[Welsh translation needed]]"
```
Comment (leoapost) [inline conf/locale/cy/LC_MESSAGES/django.po]: This change is breaking the tests

https://travis-ci.org/ministryofjustice/manchester_traffic_offences_pleas/builds/224362695#L671

I think we need to update this line:
https://github.com/ministryofjustice/manchester_traffic_offences_pleas/blob/master/apps/result/tests.py#L444

Comment (robyoung) [inline conf/locale/cy/LC_MESSAGES/django.po]: It's on it's way...

**Pull request merged**